l10n_it_fiscal_payment_term/__init__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2014 Davide Corio <davide.corio@abstract.it>
+# Copyright 2015-2016 Lorenzo Battistini - Agile Business Group
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from . import models
+
+
+def pre_init_hook(cr):
```

Comment:
Questo penso possa essere rimosso, visto che `l10n_it_fatturapa` su 11.0 non esiste

Reply:
In che senso non esiste? 

> Questo penso possa essere rimosso, visto che `l10n_it_fatturapa` su 11.0 non esiste

@eLBati  Nel senso che non è ancora stato fatto il porting? 
C'è un po' di confusione probabilmente. l10n_it_fatturapa ha tra le sue dipendenze questo modulo, ma in questo hook si controlla che l10_it_fatturapa sia installato per poi far un update sul db. 
Se questo update è necessario, è più corretto spostare questo update  in l10_it_fatturapa.
Che faccio?